pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -858,6 +858,18 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *promv1.Prome
 							"severity": "warning",
 						},
 					},
+					{
+						Alert: "VMCannotBeEvicted",
+						Expr:  intstr.FromString("kubevirt_vmi_non_evictable > 0"),
+						For:   "1m",
+						Annotations: map[string]string{
+							"description": "Eviction policy for {{ $labels.name }} (on node {{ $labels.node }}) is set to Live Migration but the VM is not migratable",
+							"summary":     "The VM's eviction strategy is set to Live Migration but the VM is not migratable",
+						},
```

Comment:
Would it make sense to attempt to succinctly explain to the admin why this is a problem, that this would cause difficulty during node drain?

Reply:
I think we do that as part of the run-book, where we include full details on the issue (including overview, mitigation, etc.) . Here we just explain the alert itself. (see for example https://github.com/kubevirt/monitoring/blob/main/runbooks/VirtAPIDown.md)
If you still think it should go here as well I can add it

Reply:
The PR for the runbook is here: https://github.com/kubevirt/monitoring/pull/6

Reply:
ok. fair point.

/lgtm